modules/microshift-ca-adding-bundle-ostree.adoc

```diff
@@ -0,0 +1,54 @@
+//Module included in the following assemblies:
+//
+//* microshift_install/microshift-embed-in-rpm-ostree.adoc
+
+
+:_content-type: PROCEDURE
+[id="microshift-ca-adding-bundle-ostree_{context}"]
+= Adding a certificate authority bundle to an ostree image
+You can include additional trusted certificate authorities (CAs) to the {op-system-ostree-first} `rpm-ostree` image by adding them to the blueprint you use to create the image. Using the following procedure sets up additional CAs to be trusted by the operating system when pulling images from an image registry.
+
+[NOTE]
+====
+The procedure requires you to configure the CA bundle customizations in the blueprint, then add steps to your kickstart file to enable the bundle. In the following steps, `data` is the key, `<value>` represents the PEM-encoded certificate.
+====
+
+.Prerequisites
+
+* Your user account has root privileges.
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the `composer-cli` tool.
+* You have root-user access to your build host.
```

Comment:
The first and the last bullet seem to be a duplicate, doesn't it?

Reply:
Yes, thanks for catching that. I will fix!